x-pack/examples/reporting_example/README.md

```diff
@@ -0,0 +1,6 @@
+Example Reporting integration!
+==============================
+
+Use this example code to understand how to add a "Generate Report" button to a Kibana page. 
```

Comment:
could we add some more description to the process here as well ?
what i would like to see mentioned is:
- that apps are responsible for displaying the page in a format we can screenshot (no controls that make no sense in the pdf)
- that apps should most likely provide something in the url to indicate its a reporting job so they can customize the rendering output

Reply:
a2ad912fae1